### DISCUSSION TITLE
Codemagic yaml refreshes after 20-30 mins

### DISCUSSION THREAD

**Discussion by bi1zi1** [codemagic-ci-cd/codemagic-docs#2680] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

I'm testing codemagic out, so I have a branch where I push updates to the yaml file.
Note: I overwrite last commit.

### Expected results

After pushing the changes to GitHub I expect the new content to show up.

### Actual results

Old yaml contents show up (no new changes).
Only after 20-30 mins the yaml file contents get refreshed.

**Comment by dtrdic** [comment, marked as answer]
hey @bi1zi1, it seems that Github had some [outages](https://www.githubstatus.com/incidents/2qxyr481ywxf) but the issue should be resolved now. Keep an eye on the status page anyway
